tasks.py

```diff
@@ -34,45 +28,69 @@ def is_truthy(arg):
     raise ValueError(f"Invalid truthy value: `{arg}`")
 
 
-PYPROJECT_CONFIG = toml.load("pyproject.toml")
-TOOL_CONFIG = PYPROJECT_CONFIG["tool"]["poetry"]
+# Use pyinvoke configuration for default values, see http://docs.pyinvoke.org/en/stable/concepts/configuration.html
+# Variables may be overwritten in invoke.yml or by the environment variables INVOKE_PYNAUTOBOT_xxx
+namespace = Collection("pyntc")
+namespace.configure(
+    {
+        "pyntc": {
+            "project_name": "pyntc",
+            "python_ver": "3.12",
+            "local": False,
+            "image_name": os.getenv("IMAGE_NAME", "pyntc"),
+            "image_ver": os.getenv("IMAGE_VER", "latest"),
```

Comment:
I'm not a fan of having essentially 2 environment variables for the same setting (e.g., `IMAGE_NAME` and `INVOKE_PYNTC_IMAGE_NAME`). Should we decommission the non-namespaced variables `IMAGE_NAME` and `IMAGE_VER`? I know it would not technically backwards compatible, but it only applies to development environments and I think we are allowed a little more lenience with backwards incompatible changes for developers.

Reply:
Thought this is a pyinvoke concept.  When you configure "namespace" the configure options automatically be migrated to `invoke_{{namespace}}_{{config setting}}` but maybe I'm wrong.  I'll look at the docs closer.

Reply:
https://docs.pyinvoke.org/en/stable/concepts/configuration.html#environment-variables

Reply:
Yes, I'm saying that you can already set it with `INVOKE_PYNTC_IMAGE_NAME` or `INVOKE_PYNTC_IMAGE_VER`. The invoke environment variables negate the need for the `os.getenv("IMAGE_NAME")` and `os.getenv("IMAGE_VER")`, so I think we should remove them to favor the ones from invoke.

```suggestion
            "image_name": "pyntc",
            "image_ver": "latest",
```

Reply:
🤦 got it